ocaml/xapi/xapi_vusb.ml

```diff
@@ -34,6 +34,13 @@ let create ~__context ~vM ~uSB_group ~other_config =
         another vusb can not create with the same USB_group. *)
     if vusbs <> [] then
       raise (Api_errors.Server_error(Api_errors.usb_group_conflict, [Ref.string_of uSB_group]));
+
+    (* USBs on different host can not be attached to one vm.*)
```

Comment:
this comment does not align with the above error message and also what's being done below.
What's the actual intention?